Catalog revision (#25)
ezsystems/commerce-documentation: docs/guide/catalog/catalog_templates.md

# Catalog templates

## Template list

The following templates are used to render the catalog and its parts:

- `Catalog/catalog.html.twig` - whole catalog
- `Catalog/product.html.twig` - individual product
- `Catalog/productType.html.twig` - product type
- `Catalog/Subrequests/listChildren.html.twig` - product categories

The templates have access to the catalog node provided by the controller.

> To show all available attributes use `getAttributeNames()`: `{{ catalogElement.attributeNames|json_encode }}`.

Reviewer: ```suggestion
To show all the available attributes, use `getAttributeNames()`: `{{ catalogElement.attributeNames|json_encode }}`.
```

### Product categories

> You can display a product category page using different layouts:

Reviewer: ```suggestion
You can use different layouts to display a product category page:
```

``` yaml
siso_core.default.category_view: product_list
```

Available options: 

- `product_list`: display product list directly
- `both`: display subcategories and product list, including an option to display facets in the left sidebar instead of the navigation

> - `categories`: display only subcategories and only on the last category (if there are no subcategories) display the product list

Reviewer: That is vague. How about

```suggestion
- `categories`: display only subcategories. For the last category only, if there are no subcategories, display the product list
```
